### PR TITLE
chore(covid): added two new rooms

### PR DESCRIPTION
**Pull request by antonyren-carleton**
Added two new rooms, rock-paper-scissors (0/0-BBA.md) and fight virus (1/1-DB.md), along with new paths to corresponding parents (0-BB and 1-D). Please update decision_tree.png once the pull request has approved :)

### CODE REVIEW

File: english/covid-19/1/1-DB.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+You have chosen to fight virus. Unfortunately, you now have been tested positive for Covid-19. You'll have to find a
```
Comment (ming-tsai) [inline english/covid-19/1/1-DB.md:1]:
```suggestion
You have chosen to fight the virus. Unfortunately, you now have been tested positive for Covid-19. You'll have to find a
```

File: english/covid-19/0/0-BBA.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Pick rock, paper, or scissors?
```
Comment (ming-tsai) [inline english/covid-19/0/0-BBA.md:1]:
```suggestion
Pick a rock, paper, or scissors?
```

**Review by ming-tsai — CHANGES_REQUESTED** [review]
Hi @antonyren-carleton,
I add some recommendation for typo

**Review by antonyren-carleton — COMMENTED** [review]
Thanks :)

**Review by antonyren-carleton — COMMENTED** [review]
Thanks for the feedback

**Review by ming-tsai — APPROVED** [review]
Hi @antonyren-carleton,
That was found XD

**Pull request merged**